praas-app-ionic/src/components/Header/index.tsx

```diff
@@ -25,7 +25,7 @@ const Header: React.FC<Props> = ({ user, logoutUser }) => {
             </IonButton>
           ) : (
             <>
-              <IonButton fill="outline" href="/signin">
+              <IonButton fill="outline" href="/login">
                 SignIn
```

Comment:
Could we change the button text too?

Reply:
Interestingly was done. But got lost in the conflict resolution